cpp/src/copying/copy.cpp

```diff
@@ -45,14 +46,7 @@ inline mask_state should_allocate_mask(mask_allocation_policy mask_alloc, bool m
  */
 std::unique_ptr<column> empty_like(column_view const& input, cudaStream_t stream)
```

Comment:
Would it make sense to remove the unused stream parameter?

Reply:
Yep. Done.